lib/v8.js

```diff
@@ -78,16 +78,24 @@ const { getOptionValue } = require('internal/options');
  * @param {string} [filename]
  * @param {{
  *   exposeInternals?: boolean,
- *   exposeNumericValues?: boolean
+ *   exposeNumericValues?: boolean,
+ *   path?: string,
+ *   encoding?: string
  * }} [options]
  * @returns {string}
  */
 function writeHeapSnapshot(filename, options) {
   if (filename !== undefined) {
     filename = getValidatedPath(filename);
   }
+
+  if (options?.path !== undefined) {
```

Comment:
Nit: I think the convention for options is generally for `!= undefined`, treating nullish values as equivalent?